base/shell/cmd/lang/en-US.rc

```diff
@@ -171,9 +171,9 @@ preset switches by prefixing any switch with - (hyphen)--for example, /-W.\n"
     STRING_DIR_HELP3 " Volume in drive %c has no label.\n"
     STRING_DIR_HELP4 " Volume Serial Number is %04X-%04X\n"
     STRING_DIR_HELP5 "\n     Total Files Listed:\n%16i File(s)% 14s bytes\n"
-    STRING_DIR_HELP6 "%16i Dir(s)% 15s bytes free\n"
+    STRING_DIR_HELP6 "%16i Dir(s)  %14s bytes free\n"
     STRING_DIR_HELP7 "\n Directory of %s\n\n"
-    STRING_DIR_HELP8 "%16i File(s)% 14s bytes\n"
+    STRING_DIR_HELP8 "%16i File(s) %14s bytes\n"
```

Comment:
Lol, so basically, this thing never worked correctly before?!

Reply:
It was working, but not consistent.